model/conversion_test.go

```diff
@@ -37,6 +37,9 @@ func TestProtoSchemaConversions(t *testing.T) {
 			{Destination: &proxyconfig.IstioService{Name: "bar"}, Weight: 75},
 			{Destination: &proxyconfig.IstioService{Name: "baz"}, Weight: 25},
 		},
+		Decorator: &proxyconfig.Decorator{
+			Operation: "foobar",
+		},
```

Comment:
I assume decorator will change the microservice view in zipkin?  Does it impact Jaeger?  Would be nice to see some screen shot of it in zipkin or others.   Should we update our sample to use this in a follow-on PR?

Reply:
It changes the span name in the zipkin data - so will be displayed in any tracing system that is able to consume that format (e.g. jaeger). I'll see what I can do about a screenshot 😄 
Yes would be good to update the example - or atleast have some RouteRules that use it. But that can be a separate PR.